CHANGELOG.md

```diff
@@ -32,6 +32,7 @@
 * [BUGFIX] Ensure that the admin client jsonnet has correct S3 bucket property. (@hedss)
 * [BUGFIX] Publish tenant index age correctly for tenant index writers. [#1146](https://github.com/grafana/tempo/pull/1146) (@joe-elliott)
 * [BUGFIX] Ingester startup panic `slice bounds out of range` [#1195](https://github.com/grafana/tempo/issues/1195) (@mdisibio)
+* [BUGFIX] Prevent writes to large traces even after flushing to disk [#1199](https://github.com/grafana/tempo/pull/1199) (@mdisibio)
```

Comment:
I think another solution that might cover more cases is returning an error from `CombineTraces()` as soon as the pieces sum up to > maxbytes. That will help fix compaction as well as querying.

Reply:
Agree, we can explore that in a separate PR.  Let's continue collecting thoughts in #1133 and #976.